zookeeper-server/src/test/java/org/apache/zookeeper/common/BaseX509ParameterizedTestCase.java

```diff
@@ -106,4 +108,24 @@ protected BaseX509ParameterizedTestCase(
         }
     }
 
+    public void init(
```

Comment:
do we need this method to be public?  protected wouldn't be more appropriate?
maybe we do need public, I might not see all the usage (it's not a small PR :p). but AFAICS we only call this from child classes.

Reply:
Fixed